### ISSUE TITLE
lambda in scope conditional causes exception

### ISSUE THREAD

**Issue by fanantoxa** [activeadmin/activeadmin#5494]
### Expected behavior
````
ActiveAdmin.register User do
  scope_to :current_user, association_method: :users_allowed_to_see
  scope 'All accepted', :is_accepted, default: true
  scope 'Not accepted', if: -> { current_user.can_see_not_accepted? } do |users|
    user.filter_some_how
  end
end
````
I expect to see `All accepted` tab. And `Not accepted` only if condition id `true`

https://activeadmin.info/3-index-pages.html
Here in doc I've find how to add condition at paragraph `Index Scopes`

### Actual behavior

````
ActionView::Template::Error (wrong number of arguments (given 1, expected 0)):
1: insert_tag renderer_for(:index)
````

If I change `->` to `proc` everything works fine

### How to reproduce

(provided code above)

**Comment by deivid-rodriguez** [comment]
@fanantoxa I fixed this issue in #5501, can you try the fix and confirm?

**Comment by fanantoxa** [comment]
@deivid-rodriguez Works good now. Thank you
